### DISCUSSION TITLE
在<template> 中使用import导入的组件能正确显示子组件的类型（比如props等），在tsx中则无法显示组件的类型？

### DISCUSSION THREAD

**Discussion by QAQLC** [vuejs/language-tools#3902] [Q&A] [answered]
如题，有没有人知道这个咋解决啊。招了issue上有关tsx相关的问题，好像没人说这个问题啊。

**Comment by so1ve** [comment, marked as answer]
Please provide a minimal reproduction

**Reply by QAQLC** [reply]
![image](https://github.com/vuejs/language-tools/assets/57053220/f7c80a93-9899-4811-8145-3579926b1780)
在template中引用子组件（HelloWord），鼠标hover在sfc子组件（HelloWolrd上，能正确显示子组件的ts推导类型
![image](https://github.com/vuejs/language-tools/assets/57053220/fcd3e7a3-94cc-4721-9d22-a7b1ed7038df)
在template中引用子组件（Child），鼠标hover在tsx子组件（Child上，能正确显示子组件的ts推导类型

![image](https://github.com/vuejs/language-tools/assets/57053220/9a903f8f-756a-4511-b56e-c95d49711805)
在tsx中引用子组件(Child)，鼠标hover在tsx子组件（Child上，不能正确显示子组件的ts推导类型
抱歉，我不会用那种在线的代码复现器，我搞了一个项目，地址是https://github.com/QAQLC/bug

**Reply by so1ve** [reply]
没事 GitHub 仓库也是可以的，不过这个行为应该是预期内的，只是显示问题
